examples/pytorch/schnet/README.md

```diff
@@ -0,0 +1,21 @@
+SchNet & MGCN
```

Comment:
Maybe also list the performance numbers of the two models on the dataset so that users have some expectation before training the model and can spot potential bugs due to the version change of packages.